src/index.ts

```diff
@@ -17,32 +19,29 @@ const config: ConnectionConfig = {
     port: Number(process.env.PGPORT),
 }
 
-const client = new Client(config)
+export const client = new Client(config)
 
-const table = client.connect()
+export const table = client.connect()
 
 
-app.use(function (req, res, next) {
-    // security 
-    //https://enable-cors.org/server_expressjs.html
-    res.header("Access-Control-Allow-Origin", "*");
-    res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
-    res.header("Access-Control-Allow-Methods", "*");
-    next();
-});
-app.use(express.json())
 
+app.use(cors())
+app.use(express.json())
+app.use('/auth', router)
 
 app.listen(PORT, () => {
     console.log(`Running on port ${PORT}`)
 })
 
 app.get('/', async function (req, res) {
     await table
-    const result = await client.query('select * from users')
-    result.rows.map(row => {
-        return res.send(row)
-    })
+    const result = await client.query('select * from users').catch(error => console.log(error))
+    if (result) {
```

Comment:
@Alexandria Is there ever a case where `result` is falsey here? Im curious what prompted this `if` check.

Reply:
I needed the `if` check because the type of `result` is inferred to be `void | QueryResult`.  I get a type error on `result.row` if I do not wrap `result` in an if block. 
The catch block is why `result` could be void. I suppose the only reason why I would need a catch is if something was wrong with the database or the table.